lib/plenario2/actions/meta_actions.ex

```diff
@@ -62,8 +65,53 @@ defmodule Plenario2.Actions.MetaActions do
     |> Repo.insert()
   end
 
-  ##
-  # update
+  @doc """
+  Get a list of column names for a `Meta` struct. 
+
+  ## Examples
+
+    iex> get_column_names(meta)
+    ["id", "location", "datetime", "observation"]
+
+  """
+  @spec get_column_names(meta :: Meta) :: list[charlist]
+  def get_column_names(meta) do
```

Comment:
These either need to go in `Plenario2.Schemas.Meta` or the `Meta.get_data_set_name` needs to be pulled out of there and added in here. Either way, I don't have a preference. Only that it's consistent.